lib/dockerfile/instruction-parser.ts

```diff
@@ -59,6 +68,17 @@ function getPackagesFromRunInstructions(
   }, {});
 }
 
+function removeRunDefFromInstruction(instruction: string) {
+  let cleanedInstruction = instruction;
+  const runDefs = ["RUN ", "/bin/sh ", "RUN /bin/sh"];
```

Comment:
Maybe again part of the old implementation but does this handle `RUN /bin/bash -l -c` ?

Reply:
It related to the tiny change to `installRegex`, see https://github.com/snyk/snyk-docker-plugin/pull/312#discussion_r548010339